src/Core/Impl/IO/FileSystem.cs

```diff
@@ -31,7 +31,8 @@ public sealed class FileSystem : IFileSystem {
         public byte[] FileReadAllBytes(string path) => File.ReadAllBytes(path);
         public void FileWriteAllBytes(string path, byte[] bytes) => File.WriteAllBytes(path, bytes);
         public Stream CreateFile(string path) => File.Create(path);
-        public Stream FileOpen(string path, FileMode mode) => File.Open(path, mode);
+        public Stream FileOpen(string path, FileMode mode) => FileOpen(path, mode);
```

Comment:
You probably didn't mean to change this to `FileOpen` from `File.Open`; this is infinitely recursive (but hasn't broken anything because it is unused).